modules/ozoneBidAdapter.md

```diff
@@ -36,8 +36,7 @@ adUnits = [{
                             publisherId: 'OZONENUK0001', /* an ID to identify the publisher account  - required */
                             siteId: '4204204201', /* An ID used to identify a site within a publisher account - required */
                             placementId: '0420420421', /* an ID used to identify the piece of inventory - required - for appnexus test use 13144370. */
-							customData": [{"settings": {}, "targeting": {"key": "value", "key2": ["value1", "value2"],}}] /* optional array with 'targeting' placeholder for passing publisher specific key-values for targeting. */                            
-							ozoneData: {"key1": "value1", "key2": "value2"}, /* optional JSON placeholder for for passing ozone project key-values for targeting. */
```

Comment:
Can you clarify on the change to remove this property `ozoneData` from the md file?  Was it intentional? 

It's still present in the adapter code from what I can tell.  It's also present on the prebid.org site (under http://prebid.org/dev-docs/bidders.html#ozone).

Reply:
Yes, this was intentional. We'll be submitting a pull request to remove it from the prebid.org documentation shortly.